python/requirements.txt

```diff
@@ -2,6 +2,9 @@
 aiocoap==0.3
 cbor2==4.0.0
 
+# Postgres for backup
+psycopg2==2.7.1
+
```

Comment:
Assuming this addition because we are using it for backup restore process. So all good 👍

Reply:
Correct, a little naughty to sneak it in with this change.